app/controllers/Application.java

```diff
@@ -53,23 +44,19 @@
 import play.mvc.Controller;
 import play.mvc.Result;
 import views.html.help.metrics.helpRuntime;
-import views.html.help.metrics.helpWaittime;
 import views.html.help.metrics.helpUsedResources;
+import views.html.help.metrics.helpWaittime;
 import views.html.help.metrics.helpWastedResources;
 import views.html.index;
-import views.html.page.comparePage;
-import views.html.page.flowHistoryPage;
-import views.html.page.helpPage;
-import views.html.page.homePage;
-import views.html.page.jobHistoryPage;
-import views.html.page.searchPage;
+import views.html.page.*;
 import views.html.results.*;
 
-import views.html.page.oldFlowHistoryPage;
-import views.html.page.oldJobHistoryPage;
-import views.html.page.oldHelpPage;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
 
-import com.google.gson.*;
+import static java.lang.StrictMath.max;
+import static java.lang.StrictMath.min;
 
 
 public class Application extends Controller {
```

Comment:
The methods in the class are very huge. Either modularize or add good java docs.